application/cypress/integration/card/card.spec.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jest/expect-expect */
+/* eslint-disable jest/valid-expect */
+/// <reference types="cypress" />
+import { registrationHelper } from '../utils/registrationHelper';
+
+describe('Card should be checked', () => {
+  it('card reducer should be empty', () => {
+    registrationHelper();
+    cy.get('[data-cy=data-cy-app-cars]').click();
+    cy.get('[data-cy=data-cy-app-BMW]').click();
+    cy.get('[data-cy=data-cy-app-Volvo]').click();
+    cy.get('[data-cy=data-cy-app-cart]').click();
+    cy.window()
+      .its('store')
+      .invoke('getState')
+      .its('cartReducer')
+      .should('have.length', 2);
```

Comment:
We shouldn't check store length for that, we should navigate to the cart page and check that we have 2 items in cart (on UI) and they rendered correctly (with correct content).

Like
1 i added bmw car to cart
2 i added volvo cart to cart
3 navigate to cart page
4 check that cart length is 2
5 first cart item is BMW
6 second cart items is volvo

Reply:
I removed, but why should* t I check reducer if all my data for mapping must be there?